### PR TITLE
Update README with Carthage install instructions

### PR DESCRIPTION
**Pull request by bchen-twilio**
Add Carthage installation instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,32 @@ To get started with the quickstart application follow these steps. Steps 1-6 wil
...
+```
+github "twilio/twilio-voice-ios"
+```
+
+Then run `carthage bootstrap` (or `carthage update` if you are updating your SDKs)
+
+On your application targets’ “Build Phases” settings tab, click the “+” icon and choose “New Run Script Phase”. Create a Run Script in which you specify your shell (ex: `/bin/sh`), add the following contents to the script area below the shell:
+
+```
+/usr/local/bin/carthage copy-frameworks
```
Comment (idelgado) [inline README.md]: do you need `/usr/local/bin` here? In other examples you reference just `carthage`

Comment (bchen-twilio) [reply README.md]: Just `carthage` actually works fine. The path instruction is actually from the [Carthage README](https://github.com/Carthage/Carthage) and adopted by Video and Voice.

File: README.md [hunk 2]
```diff
@@ -22,7 +22,32 @@ To get started with the quickstart application follow these steps. Steps 1-6 wil
...
+
+Add the following line to your Cartfile
+
+```
+github "twilio/twilio-voice-ios"
+```
+
+Then run `carthage bootstrap` (or `carthage update` if you are updating your SDKs)
+
+On your application targets’ “Build Phases” settings tab, click the “+” icon and choose “New Run Script Phase”. Create a Run Script in which you specify your shell (ex: `/bin/sh`), add the following contents to the script area below the shell:
```
Comment (piyushtank) [inline README.md]: same as objc should we show snapshots on how to do this ?

File: README.md (hunk 1 above)
Comment (idelgado) [reply README.md]: Ah. Okay may have just been an oversight by them then. I'm fine with either. I was nitpicking.

**Pull request merged**